chrometool/encoding.go

```diff
@@ -0,0 +1,252 @@
+package chrometool
```

Comment:
I'm seeing two files: chrometool/encoding.go and keybinding/encoding.go. Is this deliberate?

Reply:
Yes the encoding.go in chrometool has encodings of strings to runes found in chromedp/kb package. While keybinding/encoding has the encoding for the keycodes to readable strings